SU2_CFD/src/output/CNEMOCompOutput.cpp

```diff
@@ -317,16 +317,16 @@ void CNEMOCompOutput::LoadVolumeData(CConfig *config, CGeometry *geometry, CSolv
 
   SetVolumeOutputValue("MOMENTUM-X", iPoint, Node_Flow->GetSolution(iPoint, nSpecies));
   SetVolumeOutputValue("MOMENTUM-Y", iPoint, Node_Flow->GetSolution(iPoint, nSpecies+1));
-  SetVolumeOutputValue("VELOCITY-X", iPoint, Node_Flow->GetSolution(iPoint, nSpecies+2));
-  SetVolumeOutputValue("VELOCITY-Y", iPoint, Node_Flow->GetSolution(iPoint, nSpecies+3));
+  SetVolumeOutputValue("VELOCITY-X", iPoint, Node_Flow->GetPrimitive(iPoint, nSpecies));
```

Comment:
Could you please verify whether the current implementation actually returns the correct velocity values before you commit? With the current implementation, the velocity vector is inconsistent with the momentum vector. The primitive indices for x-, y-, and z-velocity should be `nSpecies + 2`, `nSpecies + 3`, and `nSpecies + 4` respectively for the NEMO output.